res/layout/edit_box_dialog.xml

```diff
@@ -18,19 +18,17 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:gravity="clip_horizontal"
-    android:orientation="vertical" >
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:gravity="clip_horizontal"
+              android:orientation="vertical">
```

Comment:
The only change is adding `textCapSentences` in `android:inputType="textNoSuggestions|textCapSentences"`.

> Would be good if this is not in the commit.

@tobiasKaminsky Why do you mean?